### PR TITLE
f-content-cards@v10.0.0 - Adding in changed version of content cards and updated unit tests

### PR DESCRIPTION
**Pull request by matthewhardern**
This is the start of changes to allow content cards to support multiple adapters. This PR enables the new version to work and includes unit tests, later PR's will update the docs and story files accordingly. 

- [x] Unit tests have been [created|updated]

## Browsers Tested

- [x] Chrome (latest)

### CODE REVIEW

File: packages/components/organisms/f-content-cards/src/components/_tests/ContentCards.test.js [hunk 1]
```diff
@@ -55,577 +17,175 @@ const mocks = {
...
-                            id: null,
-                            voucher: null,
-                            action: 'click',
-                            cta: button
-                        }
-                    }
-                });
-            });
-        });
+// when given 2 adapters that both return cards
```
Comment (minrwhite) [inline packages/components/organisms/f-content-cards/src/components/_tests/ContentCards.test.js:164]: I'm assuming from here and below are for implementing these tests in a follow-up?

Comment (minrwhite) [reply packages/components/organisms/f-content-cards/src/components/_tests/ContentCards.test.js:164]: Also would be good to test errored state & loading states if possible

File: packages/components/organisms/f-content-cards/src/components/ContentCards.js [hunk 2]
```diff
@@ -215,251 +96,102 @@ export default {
...
+        }, (source, error) => {
+            if (error.type === 'NoCards') {
+                this.$emit(HAS_LOADED, { adapter: source, cards: [] });
+                this.$log.info(
+                    `Content Cards Adapters (${source}) - No content cards. Key: (${this.loggingKey})`,
+                    this.logTags,
+                    {
+                        ...this.defaultLoggingData,
+                        data: {
+                            errorMessage: error
```
Comment (minrwhite) [inline packages/components/organisms/f-content-cards/src/components/ContentCards.js:177]: Might be good to add `source` as a data value into the no cards / error states as well?

**Review by minrwhite — APPROVED** [review]
Happy to approve, address other issues in a follow-up PR please

File: packages/components/organisms/f-content-cards/src/components/_tests/ContentCards.test.js (hunk 1 above)
Comment (matthewhardern) [reply packages/components/organisms/f-content-cards/src/components/_tests/ContentCards.test.js:164]: yea follow up tests, commented them as to not forget, yep will add the error and loading states

File: packages/components/organisms/f-content-cards/src/components/ContentCards.js (hunk 2 above)
Comment (matthewhardern) [reply packages/components/organisms/f-content-cards/src/components/ContentCards.js:177]: yes that is a good point will make that change in the next version

**Review by zlatinivanov-je — APPROVED** [review]

**Pull request merged**